### ISSUE TITLE
`--pull-images` flag for `eris init`

### ISSUE THREAD

**Issue by compleatang** [eris-ltd/eris-cli#112]
* eris/keys
* eris/ipfs
* eris/erisdb
* eris/eth

if `--dev`

* erisindustries/evm_compilers
* eris/compilers
* erisindustries/node
* erisindustries/python
* erisindustries/gulp
* erisindustries/embark_base
* erisindustries/sunit_base
* erisindustries/pyepm_base

**Comment by ebuchman** [comment]
lol who's using that python image?!

**Comment by zramsay** [comment]
so E pointed out that this feature may be overkill, since the default behaviour of `docker run` includes `pull`. Tests also take 2-3x as long.

**Comment by compleatang** [comment]
fair. we can add a --no-pull flag then to squelch that behaviour.

**Comment by zramsay** [comment]
`--no-pull` set to true when testing on circle (otherwise it fails)

**Comment by zramsay** [comment]
Somewhere down the line this feature got axed. I'm re-opening to suggest a `--pull-images[=false]` flag that, when true, pulls:
- `eris/base`
- `eris/keys`
- `eris/data`
- `eris/erisdb`
- `eris/epm`
because these are all needed for the pipeline from key/chain making to deploying contracts & once `init` has been run, waiting on images for each of three commands breaks the flow

**Comment by compleatang** [comment]
bumping to 0.12

**Comment by zramsay** [comment]
closing in favour of task for #309
